environment_setup/install_clion.sh

```diff
@@ -15,49 +18,55 @@ then
         echo "CLion is already installed"
         echo "================================================================"
 else
-	# Download clion
-        wget -O /tmp/CLion-${clion_version}.tar.gz "https://download-cf.jetbrains.com/cpp/CLion-${clion_version}.tar.gz"
+    # Download clion
+    wget -O /tmp/CLion-${clion_version}.tar.gz "https://download-cf.jetbrains.com/cpp/CLion-${clion_version}.tar.gz"
 
-	# Unzip and symlink to usr location
-        sudo tar xfz /tmp/CLion-${clion_version}.tar.gz -C /opt
-        sudo ln -s /opt/clion-${clion_version}/bin/clion.sh ${clion_executable_path}-${clion_version}
-        sudo ln -s ${clion_executable_path}-${clion_version} $clion_executable_path
-	
-	# Install clion desktop entry
-	wget -O ~/.local/share/applications/jetbrains-clion.desktop "https://raw.githubusercontent.com/pld-linux/clion/master/clion.desktop"
-	echo "Icon=/opt/clion-${clion_version}/bin/clion.png" >> ~/.local/share/applications/jetbrains-clion.desktop
-		
+    # Unzip and symlink to usr location
+    sudo tar xfz /tmp/CLion-${clion_version}.tar.gz -C /opt
+    
+    # Install clion desktop entry
+    wget -O ~/.local/share/applications/jetbrains-clion.desktop "https://raw.githubusercontent.com/pld-linux/clion/master/clion.desktop"
+    echo "Icon=/opt/clion-${clion_version}/bin/clion.png" >> ~/.local/share/applications/jetbrains-clion.desktop
 fi
 
+echo "================================================================"
+echo "Symlinking CLion"
+echo "================================================================"
+# Symlink clion binary to a location in the PATH
+# We do this outside the above `if` because this gives us the ability to 
+# change clion versions without re-downloading
+sudo ln -s -f /opt/clion-${clion_version}/bin/clion.sh ${clion_executable_path}-${clion_version}
+sudo ln -s -f ${clion_executable_path}-${clion_version} $clion_executable_path
+
 echo "================================================================"
 echo "Installing Bazel Plugin"
 echo "================================================================"
 
-clion_plugin_dir="${HOME}/.CLion2019.2/config/plugins"
-clion_major_version="2019.2"
-bazel_plugin_version="v2019.09.16"
+clion_plugin_dir="${HOME}/.CLion${clion_version_year}.${clion_version_major}/config/plugins"
+bazel_plugin_version="v2019.10.28"
```

Comment:
I'm assuming this newer plugin version is still compatible with CLion 2019.2.2?

Reply:
Yup, tested and using it. 2019.2.2 is still the newest version supported by the plugin